src/server.rs

```diff
@@ -220,6 +227,27 @@ impl Server {
             });
         });
 
+        let calls_count = self.calls_count.clone();
+        Python::with_gil(|py| {
+            py.allow_threads(|| {
+                thread::spawn(move || {
+                    loop {
+                        let ttl = SystemTime::now()
+                            .duration_since(UNIX_EPOCH)
+                            .unwrap()
+                            .as_secs();
+                        for item in calls_count.iter() {
+                            let mut valid_timestamps = item.value().clone();
+                            valid_timestamps.retain(|timestamp| timestamp >= &ttl);
+                            calls_count.insert(item.key().clone(), valid_timestamps);
+                            calls_count.remove_if(item.key(), |_, t| t.is_empty());
+                        }
+                        thread::sleep(Duration::from_secs(60));
```

Comment:
One question - why add a sleep for just `60` seconds? Can we add some comment for this magic number please?

Reply:
I was actually contemplating this - How often do we want to clear the cache of calls - I'm thinking if there were no calls for a minute it should cover most use cases, but maybe it should be configurable in env? WDYT?

Reply:
> but maybe it should be configurable in env? WDYT?

This approach sounds good. Would it be possible to add this config in the `robyn.env` file?